library/Vanilla/ReCaptchaVerification.php

```diff
@@ -0,0 +1,45 @@
+<?php
+/**
+ * @copyright 2009-2019 Vanilla Forums Inc.
+ * @license Proprietary
+ */
+
+namespace Vanilla;
+
+use Garden\Http\HttpClient;
+
+/**
+ * Class reCaptchaVerification
+ */
+class ReCaptchaVerification {
+
+    const RECAPTCHA_V3_URL = "https://www.google.com/recaptcha/api/siteverify";
+
+    /** @var HttpClient */
+    private $httpClient;
+
+    /**
+     * ReCaptchaVerification constructor.
+     */
+    public function __construct() {
+        $this->httpClient = new HttpClient();
+    }
+
+    /**
+     * Verify is a challenge is valid, using siteVerify endpoint.
+     *
+     * @param string $privateKey
+     * @param string $responseToken
+     * @return bool
+     */
+    public function siteVerify(string $privateKey = '', string $responseToken = ''):bool {
```

Comment:
```suggestion
    public function siteVerifyV3(string $privateKey = '', string $responseToken = ''): bool {
```

I'd recommend putting a v3 in the name, since v2 is a thing. Don't want people getting confused.

Reply:
My thinking is it should also not require the private key as a prop. It should just pull it from the config in the constructor.

Reply:
done